src/containers/Accounts/AccountAssetTab/AccountAssetTab.tsx

```diff
@@ -5,14 +5,15 @@ import { useNavigate } from 'react-router'
 import { useRouteParams } from '../../shared/routing'
 import { AccountIssuedTokenTable } from '../AccountIssuedTokenTable'
 import { AccountNFTTable } from '../AccountNFTTable/AccountNFTTable'
+import { AccountMPTTable } from '../AccountMPTTable/AccountMPTTable'
 import { ACCOUNT_ROUTE } from '../../App/routes'
 
 // TODO: Add state types or convert to react query
 interface Props {
   account: any
 }
 
-const assetTypes = ['issued', 'nft']
+const assetTypes = ['issued', 'nft', 'mpt']
```

Comment:
This should be different based on if it's the MPT Devnet or not. Here's an example PR where it was removed, if that helps: https://github.com/ripple/explorer/pull/424